src/components/Auth/Profile/tabs/ProfileTab.jsx

```diff
@@ -1,6 +1,70 @@
+import { useEffect, useState } from "react";
 import InputCom from "../../../Helpers/InputCom";
+import { UserApi } from "../../../../api/auth/user"; // Replace with actual user API function
+import { updateUser } from "../../../../api/auth/user"; // Assuming this is for updating user data
 
 export default function ProfileTab() {
+  const [user, setUser] = useState(null);
+  const [formData, setFormData] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    mobile: "",
+  });
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      try {
+        const token = localStorage.getItem("token");
+        if (!token) return;
+
+        const payload = token.split(".")[1];
+        const decodedPayload = atob(payload);
+        const decodedPayloadObject = JSON.parse(decodedPayload);
+
+        console.log(decodedPayloadObject?.id); // Log the user ID
+        const { data: userDetails } = await UserApi(decodedPayloadObject?.id); // Fetch user info
+        setUser(userDetails);
+        setFormData({
+          firstName: userDetails.firstName || "",
+          lastName: userDetails.lastName || "",
+          email: userDetails.email || "",
+          mobile: userDetails.mobile || "",
+        });
+      } catch (error) {
+        console.error("Error fetching user data:", error);
+      }
+    };
+
+    fetchUser();
+  }, []);
+
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({
+      ...prevData,
+      [name]: value,
+    }));
+  };
+
+  const handleUpdateProfile = async () => {
+    try {
+      // Add _id to the payload
+      const token = localStorage.getItem("token");
+
+      const payload = { ...formData, _id: user?._id };
+
+      console.log("Updating profile with data:", payload); // Log formData for debugging
+      const response = await updateUser(user?._id, token, payload); // Make API call to update user
+      console.log("Profile updated successfully!", response);
+    } catch (error) {
+      console.error(
```

Comment:
Do not print the error 
```suggestion
        enqueueSnackbar( "Something went wrong, Please try again later!", { variant : "error" } );
```